spring-kafka/src/main/java/org/springframework/kafka/support/serializer/ErrorHandlingDeserializer2.java

```diff
@@ -82,7 +83,11 @@
 
 	private boolean isForKey;
 
-	private BiFunction<byte[], Headers, T> failedDeserializationFunction;
+	/**
+	 * Set in order to have a supplier for a T value when deserialization fails.
+	 * @since 2.3
+	 */
```

Comment:
There is no reason in JavaDocvs on `private` properties if there is no some specific reason. 
Looks like a setter here fully covers us.